lib/matplotlib/widgets.py

```diff
@@ -611,14 +604,19 @@ def _clicked(self, event):
 
     def set_active(self, index):
         """
-        Directly (de)activate a check button by index.
-
-        *index* is an index into the original label list
-            that this object was constructed with.
-            Raises ValueError if *index* is invalid.
+        Toggle (activate or deactivate) a check button by index.
 
         Callbacks will be triggered if :attr:`eventson` is True.
 
+        Parameters
+        ----------
+        index : int
+            Index of the check button to toggle.
+
+        Raises
+        ------
+        ValueError
+            if *index* is invalid.
```

Comment:
Should we capitalize the begining of a half-sentence as well?

Reply:
sure